test/jest-e2e.json

```diff
@@ -1,9 +1,16 @@
 {
   "moduleFileExtensions": ["js", "json", "ts"],
-  "rootDir": ".",
+  "roots": ["<rootDir>/../src/", "<rootDir>"],
```

Comment:
Codacy found an issue: [Format Error: expected "\n    " ](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5718692)

Reply:
This is a minor formatting issue rather than a critical error